buildkitd/buildkitd.go

```diff
@@ -172,13 +173,21 @@ func ResetCache(ctx context.Context, console conslogging.ConsoleLogger, image, c
 // that can be used to connect to it.
 func maybeStart(ctx context.Context, console conslogging.ConsoleLogger, image, containerName, installationName string, fe containerutil.ContainerFrontend, settings Settings, opts ...client.ClientOpt) (cinfo *client.Info, winfo *client.WorkerInfo, finalErr error) {
 	if settings.StartUpLockPath != "" {
+		var tryLockDone atomic.Bool
```

Comment:
[Rob Pike has words for you](https://github.com/golang/go/issues/6123#issuecomment-66083838) O_O